docs/ftldns/regex/overview.md

```diff
@@ -0,0 +1,29 @@
+A regular expression, or RegEx for short, is a pattern that can be used for building arbitrarily complex blocking rules in *FTL*DNS.
+We implement the ERE flavor similar to the one used by the UNIX `egrep` (or `grep -E`) command.
```

Comment:
Suggestion:

We implement the POSIX Extended Regular Expressions <Link if there is one> similar...